mabwiser/mab.py

```diff
@@ -413,6 +411,68 @@ def _validate(self):
             check_true(self.n_clusters >= 2, ValueError("The number of clusters must be at least two."))
             check_true(isinstance(self.is_minibatch, bool), TypeError("The is_minibatch flag must be a boolean."))
 
+    class LSHNearest(NamedTuple):
+        """LSH Approximate Nearest Neighbors Policy.
+
+        LSHNearest is a nearest neighbors approach that uses locality sensitive hashing with a simhash to
+        select observations to be used with a learning policy.
+
+        For the simhash, contexts are projected onto an n-dimensional plane and each dimension is evaluated for its
+        sign. This is converted to a base 2 integer used as the hash value to assign the context to a hash table. This
+        process is repeated for a specified number of hash tables, where each has a unique, randomly-generated plane.
+        To select the neighbors for a context, the hash value is calculated for each hash table and any contexts with
+        the same hashes are selected as the neighbors.
+
+        As with the radius or k value for other nearest neighbors algorithms, selecting the best number of dimensions
+        and tables requires tuning. For the dimensions, a good starting point is to use the log of the square root of
+        the number of rows in the training data. This will give you sqrt(n_rows) number of hashes. The number of
+        dimensions and number of tables have inverse effects on the number of empty neighborhoods and average
+        neighborhood size.
+
+        Attributes
+        ----------
+        n_dimensions: int
+            The number of dimensions to use for each plane.
+            Integer value. Must be greater than zero.
+            Default value is 5.
+        n_tables: int
+            The number of hash tables.
+            Integer value. Must be greater than zero.
+            Default value is 3.
+        no_nhood_prob_of_arm: None or List
```

Comment:
Might be good to also explicitly mention that the probabilities are used to select an arm randomly (that's not very explicit from this explanation).